lib/src/components/NxNexusPageHeader/NxNexusPageHeader.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+import { HeaderLinkProps, Props, propTypes } from './types';
+
+export { Props };
+
+import './NxNexusPageHeader.scss';
+
+const logoImg = require('../../assets/img/logo_nexus_generic.svg');
+
+function HeaderLink({ name, href, current }: HeaderLinkProps) {
+  const classes = classnames('nx-page-header__link', {
+    'nx-page-header__link--current': current
+  });
+
+  return (
+    <a className={classes} href={href} data-text={name}>{name}</a>
+  );
+}
+
+// visible for testing
+export { HeaderLinkProps, HeaderLink };
+
+export default function NxNexusPageHeader({ links, homeLink, productInfo, children }: Props) {
+  const logoEl = <img src={logoImg} className="nx-product__logo-image" alt="⬡ Sonatype"/>,
```

Comment:
Alt text needs updating

Reply:
> Alt text needs updating

I removed it because it adds no value

Reply:
> I removed it because it adds no value

Given that it also functions as a home link, should we have some alt text that says that?

Reply:
> > I removed it because it adds no value
> 
> Given that it also functions as a home link, should we have some alt text that says that?

Whether it functions as a home link is optional (which I agree with), and any support about that should appear on the `<a>` not the `<img>`